src/main/java/mate/academy/spring/controller/MovieSessionController.java

```diff
@@ -53,7 +55,8 @@ public MovieSessionResponseDto add(@RequestBody MovieSessionRequestDto requestDt
 
     @PutMapping("/{id}")
     public MovieSessionResponseDto update(@PathVariable Long id,
-                                          @RequestBody MovieSessionRequestDto requestDto) {
+                                          @RequestBody
```

Comment:
@PathVariable add @Min(1) here Long movieId
and to every other method parameter that requests some kind of id

Reply:
Done.